src/utilities/exceptions.py

```diff
@@ -4,6 +4,10 @@
 
 
 class UserError(Exception):
+    """
+        This error should be raised every time there is a user error, e.g. invalid data formatting. 
```

Comment:
100 is a clean number, but is it 79? thats the question

Reply:
well its closer to 79 than 20. so relatively speaking, kind of.